README.md

```diff
@@ -8,16 +8,23 @@ By default, this plugin is already installed in Kuzzle.
 
 # Configuration
 
-The default configuration is:
+The default and recommended configuration is:
 
 ```json
 {
-  "algorithm": "sha256",
-  "digest": "hex"
+  "algorithm": "sha512",
+  "stretching": true,
+  "digest": "hex",
+  "encryption": "hmac"
 }
 ```
 
-All the configurations are used to set the behavior of the password hash. `algorithm` must be either `sha256` or `sha512`.
+All the configurations are used to set the behavior of the password hash.
+
+* `algorithm` must be either `sha256` or `sha512`.
```

Comment:
The complete list is available using the following command: `openssl list -digest-algorithms`
There are 40+ algorithms available, the complete list can be retrieved with `crypto.getHashes()`

Proposal change:

```suggestion
* `algorithm`: one of the supported encryption algorithms (run [crypto.getHashes()](https://nodejs.org/dist/latest-v10.x/docs/api/crypto.html#crypto_crypto_gethashes) to get the complete list). Examples: `sha256`, `sha512`, `blake2b512`, `whirlpool`, ...
```